docs/04-operation-guides/operations/evnt-01-eventing-dashboards.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Monitoring Kyma Eventing
+title: Kyma Eventing Dashboards
```

Comment:
Shall we say "Grafana Dashboards"?

Reply:
so something like `Grafana Dashboards for Kyma Eventing`?